lib/suspenders/app_builder.rb

```diff
@@ -127,7 +127,8 @@ def create_partials_directory
     end
 
     def create_shared_flashes
-      copy_file '_flashes.html.erb', 'app/views/application/_flashes.html.erb'
+      copy_file "_flashes.html.erb", "app/views/application/_flashes.html.erb"
```

Comment:
Do you want to introduce another commit normalizing all quotes in this file, rather than just this line here? Or use single quotes for this method?